src/bastion/azext_bastion/custom.py

```diff
@@ -328,6 +325,13 @@ def _is_ipconnect_request(bastion, target_ip_address):
     return False
 
 
+def handle_error_response(response):
+    errorMessage = json.loads(response.content).get('message', None)
+    if errorMessage:
+        raise ClientRequestError("Request failed with error: " + errorMessage)
+    raise ClientRequestError("Request to EncodingReservedUnitTypes v2 API endpoint failed.")
```

Comment:
please rename this error

Reply:
Will do in the next PR. Just fixing the style issues in this one